test/print_test.cpp

```diff
@@ -0,0 +1,56 @@
+#include <vector>
+
+#if COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 2
+#include <geometry_msgs/msg/point.hpp>
+#elif COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION == 1
+#include <geometry_msgs/Point.h>
+#else
+#error "Undefined or unknown COMMON_ROBOTICS_UTILITIES__SUPPORTED_ROS_VERSION"
+#endif
+
+#include <common_robotics_utilities/print.hpp>
+
+#include <gtest/gtest.h>
+
+namespace
+{
+namespace test
+{
+struct DummyType {
```

Comment:
In addition to defining a new test type here, it would be nice to use some of the existing printable types in CRU like `voxel_grid::GridIndex` in the non-ROS-message test case.

From a style perspective, `DummyType` should be
```
struct DummyType
{
  int32_t value = 0;
};
```

Reply:
Done in https://github.com/calderpg/common_robotics_utilities/pull/14/commits/3303e6861b7d7009132e66209d8eb0b6370204e1